data/persons.txt

```diff
@@ -854,37 +854,35 @@ person "Prototype B3-CC4"
 	frequency 20000
 	government "Author"
 	personality
-		forbearing mining surveillance unconstrained
+		forbearing mining surveillance unconstrained swarming
 	system
 		attributes "ember waste"
 	phrase
-		phrase
-			"friendly author"
-	phrase
-		word
-			"I may be just a piece of rock, but I know that"
-			"Even for a floating gathering of debris I know that"
-			"If there's something being unconstrained by gravity taught me, it's that"
-		word
-			" "
-		word
-			"those reactors the Navy uses are absurdly overpriced"
-			"you shouldn't buy anything the Syndicate sells. That stuff's terrible, even for their price"
-			"many reactors are in terrible conditions the moment they are sold to you. Fortunately, downloading Restock can fix this"
-			"there are lots of used engines that, after the previous owner sells them, are just refurbished enough to pass the checks and put in the local outfitter without even cleaning them a little. If I were you, I'd just download Restock and look at the new models all around the spaceport"
 		word
-			"."
-			"!"
-	phrase
-		word
-			"Long years of technological development were for this ship a pathway to many abilities some consider to be unnatural."
-			"If you ever feel useless, just remember that the people at Betelgeuse Shipyards spent months while developing the Heavy Shuttle trying to add one more gun port to it, without ever thinking about giving it more weapon space."
+			"Long years of technological development were for me a pathway to many abilities some consider to be unnatural."
 			"What were you thinking about when you hailed me? 'Oh look, an asteroid. Let's chat.'"
 			"How does it feel to try hailing an asteroid and actually succeeding?"
 			"What a day to be a non-sentient rock floating in the middle of the eternal void."
 			"When even the local asteroid belt starts chatting with you, you can tell you've been jumping a little too much lately."
 			"What's weirder, an asteroid that emits purple rays or the person that tried to talk with it?"
 			"My goals are beyond your understanding."
+			"What the Quarg say about the Builders is false. I am the only one responsible for their extinction, and I am proud of it."
+			"You think that nuclear weapons are bad? Come closer and you will see what a real weapon is."
+			"I have been classified as a mass destruction weapon by most human and alien governments. You don't want to see what I am capable of."
```

Comment:
The phrasing in English is generally "weapon of mass destruction." Is it intentional to be "mass destruction weapon"? If so, we may need a file-level comment to prevent it from being changed (and even then we may still get reports about it).

Reply:
Changed to weapon of mass destruction